### PR TITLE
Added attributes editor

### PR DESCRIPTION
**Pull request by Abyss777**
It can be easily reused, just pass `attributes` to view

### CODE REVIEW

File: web/app/view/UserDialogController.js [hunk 1]
```diff
@@ -44,5 +48,19 @@ Ext.define('Traccar.view.UserDialogController', {
             });
         }
         button.up('window').close();
+    },
+
+    attributesView: function (button) {
```
Comment (tananaev) [inline web/app/view/UserDialogController.js]: I would rename this function to `showAttributesView` to be more clear.

File: web/app/view/AttributeDialog.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Ext.define('Traccar.view.AttributeDialog', {
+    extend: 'Traccar.view.BaseEditDialog',
+
+    requires: [
+        'Traccar.view.BaseEditDialogController'
+    ],
+
+    controller: 'baseEditDialog',
+    title: Strings.stateName,
```
Comment (tananaev) [inline web/app/view/AttributeDialog.js]: I think title should be `sharedAttributes`.

**Comment by tananaev** [comment]
A couple of minor comments. Otherwise looks good.

**Comment by Abyss777** [comment]
Done minor fixes

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**